src/runtime/builtin_modules/builtins.cpp

```diff
@@ -232,8 +232,7 @@ extern "C" Box* sum(Box* container, Box* initial) {
 }
 
 extern "C" Box* id(Box* arg) {
-    i64 addr = (i64)(arg) ^ 0xdeadbeef00000003;
-    return boxInt(addr);
+    return PyLong_FromVoidPtr(arg);
```

Comment:
hmm so the reasoning behind xoring is to not confuse the conservative collector.  I would guess/hope that people don't keep id's around and especially not for objects that are dead, but it's nice to return a value that doesn't point into our heap.  Would it help if we xor'd it with a nicer value, such as `0x1000000000000000`?

You could also try out our `dump()` Python builtin:
```
>> dump(__builtins__)
Raw address: 0x12700282a8
Python object
Class: module
The '__builtin__' module
```